src/drivers/gitlab.js

```diff
@@ -176,14 +176,15 @@ class Gitlab {
     const endpoint = `/runners?per_page=100`;
     const runners = await this.request({ endpoint, method: 'GET' });
     return await Promise.all(
-      runners.map(async ({ id, name, description, active, online }) => ({
+      runners.map(async ({ id, description, active, online, status }) => ({
```

Comment:
```suggestion
      runners.map(async ({ id, description, active, online }) => ({
```
See [this conversation](https://github.com/iterative/cml/pull/653/files/fe8831ad8674554d0081ebfba0d83e4f24bf7f75#r673025416) for context.